docs/build/start-building/deploy-your-application-to-the-filecoin-testnet.md

```diff
@@ -5,12 +5,12 @@ description: We recommend all developers deploy their applications to the Fileco
 
 # Deploy your application to the Filecoin testnet
 
-If you have built your application and tested it on smaller-scale Filecoin networks (e.g. a [local network](https://docs.filecoin.io/build/start-building/interacting-with-the-network/#running-a-local-network) or the [Filecoin Nerpa network](https://docs.filecoin.io/build/start-building/interacting-with-the-network/#devnets)), you are now ready to deploy your application to the Filecoin testnet. We currently recommend all developers deploy their applications to the Filecoin testnet for many reasons, one of which is that this network is actively being monitored by the Protocol Labs team.
+If you have built your application and tested it on smaller-scale Filecoin networks (e.g. a [local network](https://docs.filecoin.io/build/start-building/interacting-with-the-network/#running-a-local-network) or the [Filecoin Nerpa network](https://docs.filecoin.io/build/start-building/interacting-with-the-network/#devnets)), you are now ready to deploy your application to the Filecoin testnet. We currently recommend _all developers_ deploy their applications to the Filecoin testnet in order to receive direct support from the Protocol Labs team and to be eligible for upcoming data storage rewards.
```

Comment:
```suggestion
If you have built your application and tested it on smaller-scale Filecoin networks, like a [local network](https://docs.filecoin.io/build/start-building/interacting-with-the-network/#running-a-local-network) or the [Nerpa devnet](https://docs.filecoin.io/build/start-building/interacting-with-the-network/#devnets), then you are now ready to deploy your application to the Filecoin testnet! We currently recommend **all developers deploy their applications to the Filecoin testnet** in order to receive direct support from the Protocol Labs team and to be eligible for upcoming data storage rewards.
```

I have a personal vendetta against parentheses, mainly because they make translation a bit of a pain.

Reply:
Got it -- this makes the first sentence super long, so I'll fix another way (without introducing parens)